src/components/Card/Card.tsx

```diff
@@ -11,18 +12,41 @@ function Card({
   flip: boolean;
   onClick: () => void;
 }) {
+  const ref = React.useRef<HTMLDivElement>(document.createElement("div"));
+
+  React.useEffect(() => {
+    let entryPoint = [0, 0];
+    const track = (moveEvent: MouseEvent) => {
+      moveEvent.stopPropagation();
+      ref.current.style.transform = `translate(${
+        (moveEvent.offsetX - entryPoint[0]) * 0.02
+      }px, ${(moveEvent.offsetY - entryPoint[1]) * 0.02}px)`;
+    };
+
+    ref.current.addEventListener("mouseenter", (event) => {
+      entryPoint = [event.offsetX, event.offsetY];
+      ref.current.addEventListener("mousemove", track);
+    });
+
+    ref.current.addEventListener("mouseleave", () => {
+      ref.current.removeEventListener("mousemove", track);
+      ref.current.removeAttribute("style");
+    });
```

Comment:
you'd need to remove `mouseenter` and `mouseleave` event listeners in the return function of the effect?